content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -25,254 +25,220 @@ Cross-cluster communication occurs over Istio Gateways of the respective cluster
+  cluster's Citadel will be configured with intermediate CA credentials
+  generated by a shared root CA. For illustration purposes, we will use a
+  pre-generated root CA certificate available as part of Istio install
+  under the `samples/certs` directory.
 
 ## Deploy Istio control plane in each cluster
 
 1. Generate intermediate CA certs for each cluster's Citadel from your
 organization's root CA. The shared root CA enables mTLS communication
-across different clusters.
+across different clusters. For illustration purposes, we will use
+the sample root certificates as the intermediate certificate.
 
-1. Create a Kubernetes secret for your generated CA certs using a command similar
-   to the following:
+1. In every cluster, create a Kubernetes secret for your generated CA certs
+   using a command similar to the following:
 
     {{< text bash >}}
+    $ kubectl create namespace istio-system
     $ kubectl create secret generic cacerts -n istio-system \
-        --from-file=path/to/intermediate/ca-cert.pem \
-        --from-file=path/to/intermediate/ca-key.pem \
-        --from-file=path/to/intermediate/root-cert.pem \
-        --from-file=path/to/intermediate/cert-chain.pem
+        --from-file=samples/certs/ca-cert.pem \
+        --from-file=samples/certs/ca-key.pem \
+        --from-file=samples/certs/root-cert.pem \
+        --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
-1. Install the Istio control plane using Helm:
+1. Install the Istio control plane in every cluster using the following commands:
 
     {{< text bash >}}
     $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
-        --set multiCluster.connectUsingGateway=true \
-        --set global.controlPlaneSecurityEnabled=true \
-        --set security.selfSigned=false > $HOME/istio.yaml
+        -f install/kubernetes/helm/istio/values-istio-multicluster-gateways.yaml > $HOME/istio.yaml
     $ kubectl create namespace istio-system
     $ kubectl apply -f $HOME/istio.yaml
     {{< /text >}}
 
 For further details and customization options, refer to the [Installation
 with Helm](/docs/setup/kubernetes/helm-install/) instructions.
 
-## Enable mTLS globally
-
-Cross-cluster communication via the Istio gateway requires mTLS.
-Therefore you must enable mTLS for all services in each cluster by
-setting the global authentication policy to permissive or stricter
-and setting a default destination rule to originate an mTLS connection
-from every pod.
+## Configure DNS
 
-For example, apply the following global configuration in each cluster:
+Service from remote clusters that are named as
+`<name>.<namespace>.global` will be resolved by the CoreDNS addon shipped with Istio. 
+In order to utilize this DNS, Kubernetes' DNS needs to be
+configured to point to CoreDNS as the DNS server for the `.global`
+dns domain. Create the following config map (or update an existing one):
 
```

Comment:
would be good to mention briefly what permission is needed to edit the kube-dns configmap.

Reply:
I dunno what they are. thats why I do this as a user initiated step - which implies user either has permissions to do this or doesn't.

Reply:
i see, thx, we can add them when we know :)